source/docs/articles/local/drush-on-pantheon.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How Drush Works on Pantheon
+description: Learn how to create a policy file and sitealias alter hook.
+keywords: Drupal drush, command line, drupal, terminus drush, cli
+---
+One thing that makes Drush site aliases so powerful is the fact that they are defined in regular PHP executable files.  This gives you many options, as the alias definitions can easily be adjusted in code, factoring out common sections, or making other adjustments directly in the alias file.  
+
```

Comment:
Can we add a statement introducing the bullet list? Maybe similar to Greg's intro:
> There are a number of reasons why one might wish to separate this code from the alias data, though.

I didn't quite understand what was being suggested until I got to the last one.